File: WebApplication/src/modules/student/models/doctoral-education.dto.ts
import { YearType } from "src/modules/application/application-one/enums/year-type.enum";
import { EducationBasicDto } from "./education-basic.dto";

export class DoctoralEducationDto extends EducationBasicDto {
  yearType: YearType;
  startDate: Date;
  finishDate: Date;
}

File: StudentCredit.Application/Students/Dtos/DoctoralEducationDto.cs
using StudentCredit.Data.Rdpzsd.Enums;

namespace StudentCredit.Application.Students.Dtos
{
    public class DoctoralEducationDto : EducationBasicDto
    {
        public YearType YearType { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? FinishDate { get; set; }
    }
}
